src/main/java/org/frc571/bradley/Constants.java

```diff
@@ -37,8 +37,8 @@
         public static final double MAX_AXIS = 1.0;
         public static final double kDeadzone = 0.3;
         public static final double kShootMotor = 0.5;
-        public static final double FIRE_COMMAND_END = 0.1;
-        public static final double FIRE_COMMAND_THRESHOLD = 0.5;
+        public static final double FIRE_COMMAND_THRESHOLD = 0.3;
+        public static final double REV_COMMAND_THRESHOLD = 0.1;
```

Comment:
This is no longer required.

Reply:
Removed FIRE_COMMAND_THRESHOLD; REV_COMMAND_THRESHOLD is needed in RobotContainer. Renamed to SHOOT_TRIGGER_THRESHOLD,